src/SDK/Trace/Tracer.php

```diff
@@ -34,7 +35,7 @@ public function spanBuilder(string $spanName): API\SpanBuilderInterface
         }
 
         if ($this->tracerSharedState->hasShutdown()) {
-            // TODO: Return a noop tracer
+            return new API\NoopSpanBuilder(Context::storage());
```

Comment:
Can you add this to the class' unit tests?

Reply:
done